.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+Thank you for your pull request. Please provide the following information.
+
+---
+
+> **Do changes in this PR impact **User Experience** (e.g., how the user interacts with Yo Office and/or the files and folders the user sees in the project that Yo Office creates)?**
+> 
+> * [ ]  Yes
+> * [ ]  No
+
+If Yes, briefly describe what is impacted.
+
+
+> **Do changes in this PR impact **documentation** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
+> 
+> * [ ]  Yes
+> * [ ]  No
+
+If Yes, briefly describe what is impacted.
+
+
+**Validation/testing performed:**
+
+Describe automated/manual testing done. 
```

Comment:
@keyur32  Let's change this to 'Describe manual testing done' since automated test will be run once the pull request is created.